src/QueuedCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Equip\Queue;
+
+final class QueuedCommand
```

Comment:
This class is required so we don't keep queuing up the same command over and over.